Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -19,6 +19,9 @@
 USE_SSL = not PARAMS.get('insecure', False)
 # Service base URL
 BASE_URL = SERVER + '/api/v1.0'
+
+VENDOR_NAME = 'Palo Alto Networks AutoFocus V2'
```

Comment:
should be the name of the integration (exact match)
```suggestion
VENDOR_NAME = 'AutoFocus V2'
```